test/test_facebook.py

```diff
@@ -31,6 +31,7 @@ class FacebookTestCase(unittest.TestCase):
     empty list for test users.
 
     """
+
```

Comment:
Having an extra space between docstrings and method code isn't a convention of this project. Even if it was, stylistic changes like this should be submitted in their own pull request.